lib/spack/spack/test/llnl/util/tty/log.py

```diff
@@ -11,10 +11,8 @@
 import sys
 import time
 
-try:
-    import termios
-except ImportError:
-    termios = None
+from typing import Optional  # novm
```

Comment:
Won't this crash when we run our tests on Python 2?

Reply:
It will not, because of the py2 directory and the typing stub module I set up in the original mypy PR.